.testcaferc.js

```diff
@@ -180,6 +180,9 @@ const config = {
     return true;
   },
   screenshots: {
+    path: 'build2/reports/junit',
+    takeOnFails: true,
+    fullPage: true,
```

Comment:
Gen3 testcafe config is set to take screenshots on failures already. This will help with debugging failed tests in CI.